lib/ruby/truffle/jruby+truffle/gem_ci/travis.txt

```diff
@@ -0,0 +1,2 @@
+activesupport
+# --git https://github.com/ruby-concurrency/concurrent-ruby.git concurrent-ruby
```

Comment:
it would fail currently, It was left there just as an example of running a gem without configuration and of a comment.